packages/web/components/table/asset-info.tsx

```diff
@@ -0,0 +1,669 @@
+    () => [
+      columnHelper.accessor((row) => row, {
+        id: "asset",
+        header: "Name",
+        cell: (cell) => (
+          <AssetCell
+            {...cell}
+            isFavorite={favoritesList.includes(cell.row.original.coinDenom)}
+            onRemoveFavorite={() =>
+              removeFavoriteDenom(cell.row.original.coinDenom)
+            }
+            onSetFavorite={() => addFavoriteDenom(cell.row.original.coinDenom)}
+          />
+        ),
+      }),
+      columnHelper.accessor((row) => row, {
+        id: "price",
+        header: () => (
+          <SortHeader
+            label={`Price (${selectedTimeFrame})`}
+            sortKey="currentPrice"
+            currentSortKey={sortKey}
+            currentDirection={sortDirection}
+            setSortDirection={setSortDirection}
+            setSortKey={setSortKey}
+          />
+        ),
+        cell: PriceCell,
+      }),
+      columnHelper.accessor((row) => row, {
+        id: "priceChart",
+        header: "",
+        cell: (cell) => (
+          <SparklineChartCell {...cell} timeFrame={selectedTimeFrame} />
+        ),
+      }),
+      columnHelper.accessor((row) => row, {
+        id: "marketCap",
+        header: () => (
+          <SortHeader
+            label="Market Cap"
+            sortKey="marketCap"
+            currentSortKey={sortKey}
+            currentDirection={sortDirection}
+            setSortDirection={setSortDirection}
+            setSortKey={setSortKey}
+          />
+        ),
+        cell: MarketCapCell,
+      }),
+      columnHelper.accessor((row) => row, {
+        id: "balance",
+        header: () => (
+          <SortHeader
+            label="Balance"
+            sortKey="usdValue"
+            currentSortKey={sortKey}
+            currentDirection={sortDirection}
+            setSortDirection={setSortDirection}
+            setSortKey={setSortKey}
+          />
+        ),
+        cell: BalanceCell,
+      }),
+      columnHelper.accessor((row) => row, {
+        id: "assetActions",
+        header: "",
+        cell: (cell) => (
+          <AssetActionsCell
+            {...cell}
+            onDeposit={onDeposit}
+            onWithdraw={onWithdraw}
+          />
+        ),
+      }),
+    ],
+    [
+      favoritesList,
+      columnHelper,
+      selectedTimeFrame,
+      sortKey,
+      sortDirection,
+      addFavoriteDenom,
+      removeFavoriteDenom,
+      onDeposit,
+      onWithdraw,
+    ]
+  );
+
+  const table = useReactTable({
+    data: assetsData,
+    columns,
+    manualSorting: true,
+    manualFiltering: true,
+    manualPagination: true,
+    enableFilters: false,
+    getCoreRowModel: getCoreRowModel(),
+    filterFns: {
+      /**
+       * these filters, even though they are not used in this table instance,
+       * are necessary to suppress errors derived by the "@tanstack/table-core"
+       * module declaration in the earn page.
+       *
+       * @fabryscript
+       */
```

Comment:
We should look into why we need this. Perhaps we can upgrade @tanstack/table

Reply:
I talked to Davide about it last week, it relates to a TypeScript module override when using the filter options. He said he will open a PR later to fix.

Reply:
Yes, it's due to tanstack table, I'll fix that in a different PR, we need to override their types. I also opened a PR to them https://github.com/TanStack/table/pull/5215

Reply:
here the PR: https://github.com/osmosis-labs/osmosis-frontend/pull/2613

Reply:
Merged stage into this branch, and removed these filters.